CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+# Contributing to Puter
+
+Welcome to Puter, the open-source distributed internet operating system. We're excited to have you contribute to our project, whether you're reporting bugs, suggesting new features, or contributing code. This guide will help you get started with contributing to Puter in different ways.
+
+<br>
+
+# Report bugs
+
+Before reporting a bug, please check our [the issues on our GitHub repository](https://github.com/HeyPuter/puter/issues) to see if the bug has already been reported. If it has, you can add a comment to the existing issue with any additional information you have.
+
+If you find a new bug in Puter, please [open an issue on our GitHub repository](https://github.com/HeyPuter/puter/issues/new). We'll do our best to address the issue as soon as possible. When reporting a bug, please include as much information as possible, including:
+
+- A clear and descriptive title
+- A description of the issue
+- Steps to reproduce the bug
+- Expected behavior
+- Actual behavior
+- Screenshots, if applicable
+- Your host operating system and browser
+- Your Puter version, location, ...
+
+Please open a separate issue for each bug you find.
+
+Maintainers will apply the appropriate labels to your issue.
+
+<br>
+
+# Suggest new features
+
+If you have an idea for a new feature in Puter, please open a new discussion thread on our [GitHub repository](https://github.com/HeyPuter/puter/discussions) to discuss your idea with the community. We'll do our best to respond to your suggestion as soon as possible.
+
+When suggesting a new feature, please include as much information as possible, including:
+
+- A clear and descriptive title
+- A description of the feature
+- The problem the feature will solve
+- Any relevant screenshots or mockups
+- Any relevant links or resources
+
+<br>
+
+# Contribute code
+
+If you'd like to contribute code to Puter, you need to fork the project and submit a pull request. If this is your first time contributing to an open-source project, we recommend reading this short guide by GitHub on [how to contribute to a project](https://docs.github.com/en/get-started/exploring-projects-on-github/contributing-to-a-project).
+
+We'll review your pull request and work with you to get your changes merged into the project.
+
+## Your first code contribution
+
+We maintain a list of issues that are good for first-time contributors. You can find these issues by searching for the [`good first issue`](https://twitter.com/HeyPuter/status/1767024945442480132) label in our [GitHub repository](https://github.com/HeyPuter/puter). These issues are designed to be relatively easy to fix, and we're happy to help you get started. Pick an issue that interests you, and leave a comment on the issue to let us know you're working on it.
```

Comment:
I don't think this twitter link is meant to be here? :laughing:

Reply:
wow!! life saver!!!